openmls/src/treesync/node/leaf_node/capabilities.rs

```diff
@@ -2,6 +2,8 @@ use openmls_traits::types::Ciphersuite;
 use serde::{Deserialize, Serialize};
 use tls_codec::{TlsDeserialize, TlsSerialize, TlsSize};
 
+#[cfg(doc)]
+use super::LeafNode;
```

Comment:
Given this is the only change in the file, is it really needed?

Reply:
Building docs is broken right now. This fixes it. `LeafNode` is used in the comment in line 14.